node-wasm/js/worker.js

```diff
@@ -1,10 +1,10 @@
-import init, { NodeWorker, NodeClient } from "lumina-node-wasm"
+import init, { NodeWorker } from "lumina-node-wasm"
 
 Error.stackTraceLimit = 99;
 
 init().then(async () => {
-    let worker = new NodeWorker(self);
-    console.log("starting worker: ", worker);
+  let worker = new NodeWorker(self);
+  console.log("starting worker: ", worker);
```

Comment:
iirc this prints some pointer, if so, maybe let's just change this to sth like
```suggestion
  console.log("Starting NodeWorker");
```